### DISCUSSION TITLE
Create New Folder for brute Force and Exhaustive Search in  Algorithm Folder

### DISCUSSION THREAD

**Discussion by sayeedajmal** [AkashSingh3031/The-Complete-FAANG-Preparation#120] [General]
Can I Create A Folder called `Brute Force and Exhaustive Search`  Because of there are A Lots of Topics I found
1. Selection Sort
2. Bubble Sort
3. Sequential Sort
4. Exhaustive Search
5. Knapsack Problem
6. Depth-First Search (DFS)
7. Breadth-First Search (BFS)
### So, Can You Give Permission for Creating Folder...
@AkashSingh3031  I am Waiting...

**Comment by AkashSingh3031** [comment]
@sayeedajmal 
You can create a file in the Algorithms folder

**Reply by sayeedajmal** [reply]
Can I write only program.. or explanation......

**Reply by AkashSingh3031** [reply]
write the program and explain in code using comment

**Reply by sayeedajmal** [reply]
Ookey...but I wanna add file like before i added .. Quickshot.md... file

**Reply by AkashSingh3031** [reply]
Ok, 1st create folder `Brute Force Algorithms` then create `.md` file in folder

**Reply by sayeedajmal** [reply]
Sir @AkashSingh3031 , Can You give the link of Badges you are added for md files... 
[like this](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/1%5D.%20DSA/2%5D.%20Algorithms/02%5D.%20Sorting%20Algorithms/Sorting_Algorithm_Explainations.md)

**Reply by AkashSingh3031** [reply]
In Which year, you?

**Reply by AkashSingh3031** [reply]
> Sir @AkashSingh3031 , Can You give the link of Badges you are added for md files...
> [like this](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/1%5D.%20DSA/2%5D.%20Algorithms/02%5D.%20Sorting%20Algorithms/Sorting_Algorithm_Explainations.md)

Insertion Sort = https://img.shields.io/badge/Insertion-Sort-brightgreen?style=for-the-badge

For others: 
https://img.shields.io/badge/NAME-COLOR?style=for-the-badge

use this like src = <link> in image tag
```
<details>
<summary><img src=link></summary>
</details>
```

**Reply by sayeedajmal** [reply]
Can You Review It 1 Time!
## Added Java Program with explanation with md file..like before! 
![Screenshot_2021-07-01_01_02_55](https://user-images.githubusercontent.com/76242275/124021137-b81df100-d9da-11eb-97f3-6c5860875656.png)

![Screenshot_2021-07-01_01_03_21](https://user-images.githubusercontent.com/76242275/124021143-b94f1e00-d9da-11eb-98fb-0ac5da19c27a.png)

**Reply by AkashSingh3031** [reply]
> Can You Review It 1 Time!
> 
> ## Added Java Program with explanation with md file..like before!
> ![Screenshot_2021-07-01_01_02_55](https://user-images.githubusercontent.com/76242275/124021137-b81df100-d9da-11eb-97f3-6c5860875656.png)
> 
> ![Screenshot_2021-07-01_01_03_21](https://user-images.githubusercontent.com/76242275/124021143-b94f1e00-d9da-11eb-98fb-0ac5da19c27a.png)

Ok

You, In which year?

**Reply by sayeedajmal** [reply]
😂😂😂😂😂😂... Okey..I am writing...another...things

**Reply by AkashSingh3031** [reply]
> 😂😂😂😂😂😂... Okey..I am writing...another...things

I am asking about your degree year because if you also in `3rd year` or `3rd year+` then don't call me sir

**Reply by sayeedajmal** [reply]
> > joyjoyjoyjoyjoyjoy... Okey..I am writing...another...things
> 
> I am asking about your degree year because if you also in `3rd year` or `3rd year+` then don't call me sir

I am In 3rd Year Diploma........
